File: Programacao-Orientada-Objeto/Variaveis-e-Fluxo-CSharp/Exercicio-6-e-9.cs
using System;

class Exercicio_6_e_9 {

    static void Main(String[] args){

        int numero;
        int [] fibonacci = new int[12];

        fibonacci[0] = 1;
        fibonacci[1] = fibonacci[0];

        numero = 0;

        while (numero >= 0){

            if (numero >= 2){

                fibonacci[numero] = fibonacci[numero - 1] + fibonacci[numero - 2];
            }

            if (fibonacci[numero] > 100) {

                break;

            }

            Console.WriteLine(fibonacci[numero]);

            numero++;
        }

    }

}

File: Programacao-Orientada-Objeto/Variaveis-e-Fluxo/Exercicio_6_e_9.java
//Exercício 6 e 9, acidentalmente usei duas variáveis pra fazer.

class Exercicio_6_e_9 {

    public static void main(String[] args){

        int numero;
        int fibonacci[] = new int[12];

        fibonacci[0] = 1;
        fibonacci[1] = fibonacci[0];
        numero = 1;

        while (numero == numero){

            if (numero == 1 || numero == 2){

                System.out.println(fibonacci[numero - 1]);

            }
            else if (numero >= 3){

                fibonacci[numero - 1] = fibonacci[numero - 2] + fibonacci[numero - 3];

                if (fibonacci[numero - 1] > 100){

                    break;

                } 

                System.out.println(fibonacci[numero - 1]);

            }

            numero++;

        }

    }

}